app/styles/app.scss

```diff
@@ -154,6 +154,47 @@
   width: 100%;
 }
 
+
+.btn-filter {
```

Comment:
If you have styling that is specific to a page, create a separate `.scss` file and import it into this one.